.travis.yml

```diff
@@ -42,7 +44,6 @@ env:
 matrix:
   allow_failures:
     - env: ROS_DISTRO=indigo PRERELEASE=true PRERELEASE_REPONAME=industrial_core PRERELEASE_DOWNSTREAM_DEPTH=0
-    - env: ROS_DISTRO=indigo PRERELEASE=true PRERELEASE_REPONAME=ros_canopen
```

Comment:
Because there's an unintuitive limitation with prerelease test feature https://github.com/ros-industrial/industrial_ci/issues/64 (I've been wanting to send a patch to the upstream but that has never occurred yet..), I kind of want to keep all prerelease jobs in `allow_failures` until it gets resolved. Thoughts?